lineman/app/components/notifications/notification_controller.coffee

```diff
@@ -0,0 +1,23 @@
+angular.module('loomioApp').controller 'NotificationController', ($scope, LmoUrlService, Records) ->
+
+
+  $scope.event = $scope.notification.event()
+  $scope.actor = $scope.event.actor()
+
+  if $scope.event.kind == 'user_added_to_group' and $scope.event.membership().inviter() == null
+    console.log 'found it', $scope.event.membership().inviterId
+
+  $scope.link = ->
```

Comment:
@gdpelican - thinking out loud that i'll move this "url for object relevant to event" into LmoUrlService.